src/components/hooks/use-visibility.tsx

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react'
+import { TUseBreakpoints } from './use-breakpoints'
+import { DerivStore } from 'store'
+
+// HINT: We will add more values to the hook in the future,
+// so for the type handling for now I'll just alias the type
+export type TVisiblity = TUseBreakpoints
+
+const useVisibility = (): TVisiblity => {
```

Comment:
@michio-rework Just a suggestion, I think something like `useScreenSize` would be a better name considering the return values of the hook.

Reply:
yeah you're right, i was struggling with the name, thanks :pray: